### PR TITLE
feat(profiler_builtin): add enable/disable interface

### PR DESCRIPTION
**Pull request by FASTSHIFT**
### Description of the feature or fix

Support temporarily shutting down profiler management.

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html)
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

File: src/misc/lv_profiler_builtin.c
Comment (PGNetHun) [inline src/misc/lv_profiler_builtin.c:96]: If you set it to `true` in init, then it's worth to set it `false` in uninit function.

**Review by PGNetHun — CHANGES_REQUESTED** [review]

File: src/misc/lv_profiler_builtin.c
Comment (FASTSHIFT) [reply src/misc/lv_profiler_builtin.c:96]: The structure will be [cleared](https://github.com/lvgl/lvgl/blob/master/src/misc/lv_profiler_builtin.c#L103) in `lv_profiler_builtin_uninit`, which will also set enable to false.

**Review by PGNetHun — APPROVED** [review]

**Review by kisvegabor — APPROVED** [review]
I'll merge it after the release.

**Comment by kisvegabor** [comment]
Hit the merge button by accident... :sweat:

**Pull request merged**
